src/service/attach_response.proto

```diff
@@ -0,0 +1,37 @@
+syntax = "proto3";
+package helium.mapper;
+
+import "service/gps_response.proto";
+
+message attach_response_t {
```

Comment:
Can we drop the `_t` convention throughout this PR?

Reply:
Yeah, I liked it at first because it made conventional C-named struct but it's gross elsewhere.